pkg/virt-handler/cgroup/cgroup_v2_manager.go

```diff
@@ -65,3 +72,43 @@ func (v *v2Manager) GetCgroupVersion() CgroupVersion {
 func (v *v2Manager) GetCpuSet() (string, error) {
 	return getCpuSetPath(v, "cpuset.cpus.effective")
 }
+
+func (v *v2Manager) CreateChildCgroup(name string, subsystem string) (bool, error) {
+	subsyspath, err := v.GetBasePathToHostSubsystem(subsystem)
+	if err != nil {
+		return false, err
+	}
+
+	newgrouppath := filepath.Join(subsyspath, name)
+	if _, err = os.Stat(newgrouppath); !os.IsNotExist(err) {
+		return true, nil
+	}
+
+	// Write "+subsystem" to cgroup.subtree_control
+	wval := "+" + subsystem
+	err = runc_cgroups.WriteFile(subsyspath, "cgroup.subtree_control", wval)
+	if err != nil {
+		return false, err
+	}
+
+	// Create new cgroup directory
+	err = util.MkdirAllWithNosec(newgrouppath)
+	if err != nil {
+		log.Log.Infof("mkdir %s failed", newgrouppath)
+		return false, err
+	}
+
+	// Enable threaded cgroup controller
+	err = runc_cgroups.WriteFile(newgrouppath, "cgroup.type", "threaded")
+	if err != nil {
+		return false, err
+	}
+
+	// Write "+subsystem" to newcgroup/cgroup.subtree_control
+	wval = "+" + subsystem
+	err = runc_cgroups.WriteFile(newgrouppath, "cgroup.subtree_control", wval)
+	if err != nil {
+		return false, err
+	}
```

Comment:
One would just pass a list of controllers to be enabled, rather than a single controller. However, for the 
current feature, one controller is sufficient. So i think one could extend support to multiple controllers when the need arises.